### ISSUE TITLE
subversion: Download failed: https://serf.googlecode.com/svn/src_releases/serf-1.3.8.tar.bz2

### ISSUE THREAD

**Issue by timotheecour** [Linuxbrew/homebrew-core#429]
using linuxbrew:
```
brew install subversion

Hunk #1 succeeded at 26150 (offset 784 lines).
patching file subversion/bindings/swig/perl/native/Makefile.PL.in
patching file build/get-py-info.py
==> Downloading https://serf.googlecode.com/svn/src_releases/serf-1.3.8.tar.bz2

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "subversion--serf"
Download failed: https://serf.googlecode.com/svn/src_releases/serf-1.3.8.tar.bz2
```

after 'brew edit subversion' and 
s/
https://serf.googlecode.com/svn/src_releases/serf-1.3.8.tar.bz2
/
https://archive.apache.org/dist/serf/serf-1.3.8.tar.bz2

fails with:
```
scons: Reading SConscript files ...
/bin/sh: 1: /usr: Permission denied
OSError: '/usr --cflags gssapi' exited 126:
  File "/tmp/subversion--serf-20160616-10941-bmo7e4/serf-1.3.8/SConstruct", line 360:
    env.ParseConfig('$GSSAPI --cflags gssapi')
  File "/home/timothee/.linuxbrew/Cellar/scons/2.4.1/libexec/scons-local/SCons/Environment.py", line 1576:
    return function(self, self.backtick(command))
  File "/home/timothee/.linuxbrew/Cellar/scons/2.4.1/libexec/scons-local/SCons/Environment.py", line 594:
    raise OSError("'%s' exited %d" % (command, status))
```

**Comment by sjackman** [comment]
Please report `brew gist-logs subversion`

**Comment by timotheecour** [comment]
https://gist.github.com/timotheecour/1712a8e9a6102b55e515936dcdc15a0d

**Comment by sjackman** [comment]
A few possible work arounds are to install Linuxbrew in `/home/linuxbrew/.linuxbrew` which will allow you to use the precompiled binary bottle of `subversion`. The second is to `brew install subversion --without-serf`.  If you want to try fixing this yourself, I believe the line `GSSAPI=/usr` has to be changed for Linuxbrew to point to the location of the `krb5` installed by Linuxbrew. Try `brew edit subversion`

**Comment by syarra** [comment]
I have submitted a PR (#475) to try to address this issue.

**Comment by sjackman** [comment]
Fixed by #475.

**Comment by rwhogg** [comment]
@syarra thanks a bunch for fixing this, it's been annoying me for a while now :smile:
